packages/documentation/copy/en/handbook-v2/Modules.md

```diff
@@ -404,3 +404,57 @@ You can see all of the available options and what their emitted JavaScript code
 ## TypeScript namespaces
 
 TypeScript has its own module format called `namespaces` which pre-dates the ES Modules standard. This syntax has a lot of useful features for creating complex definition files, and still sees active use [in DefinitelyTyped](/dt). While not deprecated, the majority of the features in namespaces exist in ES Modules and we recommend you use that to align with JavaScript's direction. You can learn more about namespaces in [the namespaces reference page](/docs/handbook/namespaces.html).
+
+## package.json
+
+To use modules your `package.json` must include `"type": "module",`. For example,

```

Comment:
This looks to be covered by https://www.typescriptlang.org/docs/handbook/esm-node.html, right? Maybe we just need links if we don't have them already?